media/migrations/0003_adding_track_id.py

```diff
@@ -0,0 +1,36 @@
+from django.db import migrations, models
+import requests
+import re
+
+
+
+def derive_track_id(apps, schema_editor):
+    SoundCloudAsset = apps.get_model('media', 'SoundCloudAsset')
+
+    for asset in SoundCloudAsset.objects.all():
```

Comment:
some might already have `track_id`...

```suggestion
    for asset in SoundCloudAsset.objects.filter(track_id=''):
```

Reply:
I thought about doing it first, but we are running migration - this field wasn't event there before and 0003 runs right after 0002, so they all will be "", no?

Reply:
Or it's more of a safety measure?

Reply:
in this case not an issue but imagine if you are a heavy load site, if you deploy without downtime there is a chance that some users already might insert a record here. just a safety thing which is a good practice in migrations. also this effectively does a table scan with locks for each record. could be race conditions, etc. better to minimize the risk as much as possible.

having said all that, yes this is not an issue for this particular case. feel free to merge as is